app/models/grda_warehouse/hud/site.rb

```diff
@@ -7,6 +7,7 @@
 module GrdaWarehouse::Hud
   class Site < Base
     include HudSharedScopes
+    include ::HMIS::Structure::Base
     acts_as_paranoid column: :DateDeleted
```

Comment:
Should this use `hud_paranoid_column`?

Reply:
It could.  This class is deprecated, so the fewer dependencies the better.